data/mods/Discrete_Batteries/items.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "magazine_battery_mod",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_green",
+    "ammo": "battery",
+    "name": "battery compartment mod",
+    "description": "A battery compartment mod that allows using high-density, rechargeable vehicle batteries in regular tools.",
+    "price": 2000,
+    "material": ["plastic", "aluminum"],
+    "weight": 500,
+    "volume": 1,
+    "category" : "spare_parts",
+    "use_action": "MAGAZINE_BATTERY",
+    "magazines": [ [
```

Comment:
Since you wrote this PR there now is `magazine_adaptor` which the beltfeed item uses